src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -0,0 +1,375 @@
+}
+
+.accordionItem {
+  cursor: pointer;
+  &:not(:last-child) {
+    margin-bottom: var(--Space-24);
+    @include for-tablet-only {
+      margin-bottom: var(--Space-32);
+    }
+  }
+  &.active {
+    .accordionChildren {
+      display: block;
+    }
+    .accordionParent svg {
+      transform: scale(1, -1);
+    }
+  }
+  .accordionParent > div:first-child,
+  .accordionChild {
+    max-width: calc(100% - 14px);
+    padding-right: var(--Space-64);
+    @include for-tablet-only {
+      padding-right: 120px;
+    }
+  }
+}
+
+.accordionParent {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  > svg {
+    width: 12px;
+    height: 12px;
+    margin-bottom: var(--Space-8);
+  }
+}
+
+.accordionChildren {
+  display: none;
+  padding-bottom: var(--Space-8);
+}
+
+.accordionChild {
+  padding-top: var(--Space-24);
+}
+
+.belowAccordion {
+  display: flex;
+  flex-direction: column;
+  .belowAccordionItem {
+    display: inline-block;
+    &:not(:last-child) {
+      margin-bottom: var(--Space-24);
+      @include for-tablet-only {
+        margin-bottom: var(--Space-32);
+      }
+    }
+  }
+}
+
+.dropdownNav {
+  display: flex;
+  .dropdownNavParent {
+    cursor: pointer;
+    height: 64px;
+    display: flex;
+    align-items: center;
+    padding-top: 4px;
+    border-bottom: 4px solid transparent;
+    &:hover {
+      border-bottom-color: var(--BrandForest);
+
+      > div:first-child {
+        /** either this text shadow or we have to set fixed widths on each nav item
+          * in order to meet bolding spec without changing item width
+        **/
+        text-shadow: 0 0 0.25px #272727, 0 0 0.25px #272727;
+      }
+      .dropdownNavChildren {
+        display: block;
+        cursor: default;
+      }
+    }
+    &:not(:last-child) {
+      padding-right: var(--Space-12);
+    }
+    &:not(:first-child) {
+      padding-left: var(--Space-12);
+    }
+    .dropdownNavParentIcon {
+      margin-left: 4px;
+    }
+  }
+}
+
+.dropdownNavChildren {
+  display: none;
+  position: absolute;
+  top: 64px;
+  left: 0;
+  width: 100%;
+  background: var(--White);
+  border-top: solid 1px var(--GrayStrokeAndDisabled--translucent);
+}
+
+.dropdownNavChild {
+  margin-bottom: 16px;
+  a {
+    text-decoration: none;
+    &:hover {
+      opacity: 0.78;
+    }
+  }
+}
+
+.dropdownNavChildrenInner {
+  display: flex;
+  padding: var(--Space-40) 0 var(--Space-24);
+  .dropdownNavChildrenCta {
+    &:hover {
+      span {
+        opacity: 0.78;
+        text-decoration: underline;
+      }
+    }
+    margin-right: 120px;
+    @include for-laptop-only {
+      max-width: 264px;
+    }
+    @include for-desktop-only {
+      max-width: 320px;
+    }
+    svg {
+      margin-left: 4px;
+      width: 19px;
+      height: 19px;
+    }
+  }
+  .dropdownNavChildrenItems {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    max-width: 504px;
```

Comment:
why 504px that's an odd value I think

Reply:
bespoke from Figma layout, but yes it's a bit odd, i'll see if i can sort a more readable solution with padding/margins